packages/formvuelate/src/utils/Helpers.js

```diff
@@ -57,15 +57,14 @@ export const forEachSchemaElement = (schema, fn, path = '') => {
   const normalizedSchema = normalizeSchema(unref(schema))
 
   for (const row of normalizedSchema) {
+    let rowPath = path
```

Comment:
Let's add some spacing here please :)
```suggestion
    let rowPath = path
    
```